### ISSUE TITLE
bug: ion-content will not scroll after content update triggered from ion-menu (iOS only)

### ISSUE THREAD

**Issue by JayBizzle** [ionic-team/ionic-framework#23061]
# Bug Report

**Ionic version:**

[ ] **4.x**
[x] **5.x**

**Current behavior:**
After triggering a content update inside `ion-content` from an `ion-menu` sidemenu, the `ion-content` will become unscrollable

**Expected behavior:**
Should be able to scroll after the content update

**Steps to reproduce:**

- Clone this repo - https://github.com/JayBizzle/ionic-menu-scroll-bug
- `npm i`
- `ionic capacitor run ios --device`
- Run in an iPhone simulator from XCode (i'm using iOS 14.4 in an iPhone 12)
- When the initial pages loads, you will be able to scroll
- Open the menu
- Click `Update content`
- Close the menu
- You will now not be able to scroll

Now, if you reopen the menu after updating the content, then immediately close the menu, the scrolling will start working again.

The same is true if you inspect the page via Safari debug, and just update ANY CSS property on ANY element.

It seems like triggering a redraw on the page somehow makes the scrolling start working again.

It will stop working again after another content update.

This works as expected on an Android device and in a desktop browser

**Other information:**

**Ionic info:**

```
Ionic:

   Ionic CLI       : 6.13.1 (/Users/markbeech/.nvm/versions/node/v12.18.3/lib/node_modules/@ionic/cli)
   Ionic Framework : @ionic/vue 5.6.0

Capacitor:

   Capacitor CLI   : 2.4.7
   @capacitor/core : 2.4.7

Utility:

   cordova-res : not installed
   native-run  : not installed

System:

   NodeJS : v12.18.3 (/Users/markbeech/.nvm/versions/node/v12.18.3/bin/node)
   npm    : 7.6.0
   OS     : macOS Big Sur
```

**Comment by JayBizzle** [comment]
Okay, so on further investigation, I have a hack that can get this to work as expected.

On the `ion-menu` we added a listener to the `ionDidClose` event..

```html
<ion-menu @ionDidClose="iOSScrollFixHack()" side="end" type="overlay" content-id="main-content" menu-id="search">
  //
</ion-menu>
```

Then, inside our `ion-content` we have just placed a dummy element like this...

```html
<ion-content :fullscreen="true">
  // All our content here

  <div id="ios-search-scroll-fix-hack"></div>
</ion-content>
```

Finally, we have this function that is triggered when the menu has been closed

```javascript
const iOSScrollFixHack = () => {
  document.getElementById('ios-search-scroll-fix-hack').style.opacity = "0.9";
}
```

We tried this with `ionWillClose` but that doesn't work.

Hope that helps. I will report anything else we discover.

Thanks 👍

**Comment by liamdebeasi** [comment]
Thanks for the issue. I am going to close this as a duplicate of https://github.com/ionic-team/ionic-framework/issues/22306. 

This is a bug in WebKit and will be fixed in iOS 14.5: https://github.com/ionic-team/ionic-framework/issues/22306#issuecomment-771705291
